website/content/docs/api-gateway.mdx

```diff
@@ -0,0 +1,171 @@
+---
+layout: docs
+page_title: API Gateway
+description: Using Consul API gateway functionality
+---
+
+# Consul API Gateway
+
+This topic describes how to use the Consul API Gateway module, which enables applications and services running in a datacenter to be accessed by network clients outside the datacenter. This type of network traffic is commonly referred to as "north-south" network traffic.
+
+## Introduction
+
+Consul API Gateway is an implementation of the Kubernetes Gateway [API Specification](https://gateway-api.sigs.k8s.io/). The specification defines a set of custom resource definitions (CRD) that can create logical gateways and routes.
+
+## Requirements
+
+Your datacenter must meet the following requirements prior to configuring the Consul API Gateway:
+
+- A Kubernetes cluster must be running
+- Consul 1.11.1+
+
+## Installation
+
+1. Create a values file for your Consul server agents that contains the following parameters:
+
+   ```yaml
+   global:
+     name: consul
+     image: 'hashicorp/consul:1.11.1'
+     tls:
+       enabled: true
+   connectInject:
+     enabled: true
+   controller:
+     enabled: true
+   ```
+
+1. Install Consul API Gateway using the standard Consul Helm chart and specify the custom values file.
+
+```shell-session
+$ helm install consul hashicorp/consul --version 0.39.0 -f values.yaml
+```
+
+The following components will be installed: - Gateway controller - CRDs required by the Kubernetes Gateway API specification - `kustomize` manifests for completing the installation
+
+1. After `helm` installs Consul API Gateway packages, issue the following commands to apply the API gateway to your Kubernetes cluster.
```

Comment:
This CRD creation and deployment step may be integrated into the Helm chart before the release [Slack thread](https://hashicorp.slack.com/archives/C01RWVBQ6GJ/p1642000326061100).. if it is, then these step can be completely removed from your docs and the Learn guide.

Reply:
@nickethier Is this something you'd be able to update with the Helm install instructions?